src/api/ledger/parse/suspended-payment-finish.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const sjclcodec = require('sjcl-codec');
+const utils = require('./utils');
+
+function convertHexToString(hexString) {
+  const bits = sjclcodec.hex.toBits(hexString);
+  return sjclcodec.utf8String.fromBits(bits);
+}
+
+function parseSuspendedPaymentFulfillment(tx: Object): Object {
```

Comment:
Need to be consistent: "SuspendedPaymentFinish" vs. "SuspendedPaymentFulfillment". I think "Fulfillment" or "Execution" are both better than "Finish".